src/decoder.rs

```diff
@@ -1356,16 +1384,16 @@ mod tests {
 
     fn accepts(d: &Decoder, input: &[u8], tail: &[u8], expect: Value) {
         let program = Program::new();
-        let mut scope = Scope::new();
-        let (val, remain) = d.parse(&program, &mut scope, ReadCtxt::new(input)).unwrap();
+        let scope = Scope::Empty;
```

Comment:
This should be simple enough to inline into its single site of use, though admittedly not a huge need to do so.

Reply:
Fixed.